components/dashboard/src/settings/settings-menu.ts

```diff
@@ -18,7 +18,7 @@ export default [
         link: ['/plans']
     },
     {
-        title: 'Teams',
+        title: 'Team Plans',
```

Comment:
**thought**: Alternatively, this could read _Subscriptions_.

Reply:
So, the reason we initially went with "Plans" and not "Subscriptions" for the individual "Plans" page is:

- An individual has only one Subscription with Gitpod (paid or not), but that Subscription can be upgraded/downgraded to various Plans (e.g. Open Source, Professional, Unleashed, etc)

So the page could have been called "Subscription" (singular) or "Plans" (plural). We felt "Plans" more appropriate, because that's exactly what this page shows you (a collection of available Plans).

For Teams, this is a bit more awkward, because you can possibly have up to three Team Subscriptions (one for each Team Plan).

I'm a bit more in favour of "Team Plans" (to be consistent with "Plans"), but I agree that this could also become "Team Subscriptions".